dev-tools/startup/discovery-provider.sh

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env sh
 
+if [[ "$audius_db_url" == "" ]]; then
+    export audius_db_url="postgresql+psycopg2://postgres:postgres@db:5432/discovery_provider_${replica}"
+    export audius_db_url_read_replica="postgresql+psycopg2://postgres:postgres@db:5432/discovery_provider_${replica}"
+fi
```

Comment:
how does this work with the start script [here](https://github.com/AudiusProject/audius-protocol/blob/main/discovery-provider/scripts/start.sh#L77-L86)? I think the start script part can be removed

Reply:
This is local dev related, that is the general discovery startup, we should keep both

Reply:
These scripts are unrelated